### DISCUSSION TITLE
Report Abuse REPO

### DISCUSSION THREAD

**Discussion by dugulingping** [community/community#41475] [Code Security] [answered]
### Select Topic Area

Question

### Body

Dear Github Team,

I am writing to report the repo and own repos user of [https://github.com/cirosantilli/china-dictatorship](https://github.com/cirosantilli/china-dictatorship) in our community. Their actions have been causing harm to our community and are in violation of Chinese national laws.

Specifically, they have been polluting search results, disrupting the harmony and development of our community, engaging in illegal activities, causing harm to others, and damaging community property.

This project is politically sensitive. As a technology sharing website, github should not be used as a political propaganda tool for some people. There is a lot of bloody, violent and fake content inside the project, which has interfered with me and many other users, and made it difficult for us to use github produces a bad experience.

This behavior is unacceptable and must be addressed. I urge you to take appropriate action to address this issue and prevent further harm to our community.

Thank you for your attention to this matter.

Sincerely,
Github's user

**Comment by queenofcorgis** [comment, marked as answer]
Hi @dugulingping 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms.  I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
